### ISSUE TITLE
About training 

### ISSUE THREAD

**Issue by a41888936** [MichalBusta/E2E-MLT#4]
Hi, Thanks for shraring amazing code.

I have some question and ask some favor.

I run demo using your pre-trained model.

It works fine, but sometimes it recognized as a wrong language.( real: Korean word,  pred: Chinese word)

So, I try to make my own model for two languages(Korean, English )

In your README, you shared Synthetic MLT Data, but there isn't English dataset.

> 1. Can you also share English dataset like other language?

I know there is English data (http://www.robots.ox.ac.uk/~vgg/data/scenetext/)

Honestly ,I have no idea how to pre-processing for training.

So If you share your English dataset, It would be very helpful for me.

And I have a question about the loss value in the training process.

`epoch 29[297400], loss: -0.740, bbox_loss: 1.441, seg_loss: -1.745, ang_loss: 0.142, ctc_loss: inf, rec: 0.53351 in 0.008`

ctc_loss looks weird. but when I run the demo using my trained model, It works fine though.

> 2. I want to make sure whether I use dataset properly or not.

Here is my sample.

1. "train_list" in train.py

![ant hill_102_1](https://user-images.githubusercontent.com/24805731/48389713-1cb0b280-e742-11e8-98c1-bcfc87ec0db6.jpg)

```
4 0.6138464864095052 0.9437334566510797 0.100082146022 0.0380848404567 -0.0165453118083 승인
4 0.7581702677408855 0.9419909103653437 0.15256916863 0.0400553852073 -0.0165452067969 신청을
4 0.09844179789225262 0.4498745809216279 0.127543301498 0.0471818991097 0.0601694282711 킬러와
4 0.14116963704427085 0.934938899501977 0.153929254901 0.0502906237088 -0.00324781919693 클락슨은
4 0.30354619344075523 0.9330855692100061 0.0971145094056 0.0515150747853 -0.0032484308943 같은
4 0.127603022257487 0.5502413635996427 0.106233448994 0.0524479823728 0.0817744338364 인수를
4 0.1683737055460612 0.8213418556825959 0.105878256893 0.0450125720438 -0.0295469131212 수요와
4 0.7358302307128907 0.8595470419185295 0.195197484112 0.0518889909319 -0.0354592041309 갑이다
4 0.5390819803873699 0.710513298238861 0.090994413401 0.0300411324556 0.395192956606 우리당은
4 0.04096550623575847 0.7650540020053984 0.0663619143835 0.0345938411181 -0.0283670735217 2005
4 0.1388903299967448 0.7581292590955748 0.0735245075651 0.0328411962243 -0.0289554923933 03:05
4 0.1029958724975586 0.311124815557995 0.159051668836 0.0493272034958 0.0643214178445 아니라
```

2. "ocr_feed_list" in train.py

![word_27583](https://user-images.githubusercontent.com/24805731/48389902-cf811080-e742-11e8-8157-b4db5d4519da.png)

My question is so basic since I'm a newbie in this filed. Sorry for that(also my bad English).

**Comment by alwc** [comment]
I also have a few questions regarding the training data

1. In `sample_train_data/MLT/done/gt_img_5407.txt`, I assume it is in `x1,y1,x2,y2,x3,y3,x4,y4,script,transcription` format, but how come in `sample_train_data/MLT/icdar-2015-Ch4/Train/voc_img_784.txt` it doesn't have coordinates at all?

2. For the multi-lingual dataset, given one of @a41888936 example: 

> 4 0.6138464864095052 0.9437334566510797 0.100082146022 0.0380848404567 -0.0165453118083 승인

Is it `script, left, bottom, width, height, angle, transcript`? Do you have the script that convert the synthtext format into icdar2015? Thanks!

**Comment by MichalBusta** [comment]
> 1. In `sample_train_data/MLT/done/gt_img_5407.txt`, I assume it is in `x1,y1,x2,y2,x3,y3,x4,y4,script,transcription` format, but how come in `sample_train_data/MLT/icdar-2015-Ch4/Train/voc_img_784.txt` it doesn't have coordinates at all?

sorry, wrong file - fixed now.

> 2. For the multi-lingual dataset, given one of @a41888936 example:
> 
> > 4 0.6138464864095052 0.9437334566510797 0.100082146022 0.0380848404567 -0.0165453118083 승인
> 
> Is it `script, left, bottom, width, height, angle, transcript`? Do you have the script that convert the synthtext format into icdar2015? Thanks!

the data loader supports both formats: 
 - https://github.com/MichalBusta/E2E-MLT/blob/master/data_gen.py#L615

**Comment by MichalBusta** [comment]
> > 1. Can you also share English dataset like other language?

done. 

> And I have a question about the loss value in the training process.
> 
> `epoch 29[297400], loss: -0.740, bbox_loss: 1.441, seg_loss: -1.745, ang_loss: 0.142, ctc_loss: inf, rec: 0.53351 in 0.008`
> 
> ctc_loss looks weird. but when I run the demo using my trained model, It works fine though.
> 
data looks ok. inf CTC loss can be in case when output sequence length is smaller than transcription length. 
or: https://github.com/baidu-research/warp-ctc/issues/23

**Comment by a41888936** [comment]
@MichalBusta Oh, That was really helpful. Thanks :)

**Comment by ZDDEAN** [comment]
> > 1. 在`sample_train_data/MLT/done/gt_img_5407.txt`，我认为它是`x1,y1,x2,y2,x3,y3,x4,y4,script,transcription`格式的，但`sample_train_data/MLT/icdar-2015-Ch4/Train/voc_img_784.txt`它怎么没有坐标？
> 
> 对不起，错误的文件 - 现在修好了。
> 
> > 1. 对于多语言数据集，给出一个@ a41888936示例：
> > 
> > > 4 0.6138464864095052 0.9437334566510797 0.100082146022 0.0380848404567 -0.0165453118083승인
> > 
> > 
> > 是`script, left, bottom, width, height, angle, transcript`吗？您是否有将synthtext格式转换为icdar2015的脚本？谢谢！
> 
> 数据加载器支持两种格式：
> 
> * https://github.com/MichalBusta/E2E-MLT/blob/master/data_gen.py#L615

请问如何能分享一下synthtext格式转换为icdar格式的脚本吗，谢谢鸭
